python/ray/dashboard/client/src/api.ts

```diff
@@ -124,8 +124,10 @@ export interface RayletInfoResponse {
         }
       | {
           actorId: string;
+          actorTitle: string;
           requiredResources: { [key: string]: number };
           state: -1;
+          invalidStateType?: 'infeasibleActor' | 'waitUntilResourceAvailable';
```

Comment:
Let's make the naming a bit more consistent here, how about 'pendingActor' instead of 'waitUntilResourceAvailable' (similar in the rest of the code)

Reply:
Agree! I changed the name to `pendingActor`